intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -87,6 +91,32 @@ class Model {
     generate_count = 0;
   }
 
+  static size_t jblas_qpack(const int8_t* src_w, const float* src_scales, const int8_t* src_zps, void* dstpr,
+                            const quant_params_internal params, int nthread, int n, int k);
+  static size_t jblas_quantize(const float* src_w, void* dstpr, const quant_params_internal params, int nthread, int n,
+                               int k);
+  static size_t np_jblas_qpack(py::array_t<int8_t> src_w, py::array_t<float> src_scales, py::array_t<int8_t> dst) {
+    int8_t* w_ptr = src_w.mutable_data();
+    float* scales_ptr = src_scales.mutable_data();
+    int8_t* dst_ptr = dst.mutable_data();
+
+    quant_params_internal q_params;
+    q_params.bits = quant_bits::q4;
+    q_params.scale_dtype = quant_sdtype::fp32;
+    q_params.compute_dtype = quant_comp::int8;
+    q_params.group_size = 128;
+    return Model::jblas_qpack(w_ptr, scales_ptr, nullptr, dst_ptr, q_params, 1, src_w.shape(0), src_w.shape(1));
```

Comment:
if leave to next pr, this should be removed or renamed?

Reply:
I will add jblas soon